src/components/App/App.jsx

```diff
@@ -14,9 +14,9 @@ const App = () => {
 
   return (
     <div className="app">
-      <AppHeader/>
+      <AppHeader />
       <Bookings />
```

Comment:
I believe the `<Restaurant />` component was supposed to be placed here, below Bookings, according to the instructions provided in the issue.

Reply:
Thank you very much @PERicci, I made changes in App.jsx as per your feedback.